rmtest/__init__.py

```diff
@@ -106,6 +106,19 @@ def assertExists(self, r, key, msg=None):
     def assertNotExists(self, r, key, msg=None):
         self.assertFalse(r.exists(key), msg)
 
+    def assertInitArgsFail(self):
+        try:
+            c, s = self.client, self.server
+        except Exception:
+            delattr(self, '_server')
+            self.assertOk('OK')
+        else:
+            self.assertOk('NotOK')  
+
+    def assertInitArgsSuccess(self):
+        c, s = self.client, self.server
+        self.assertOk('OK', self.cmd('set', 'test', 'foo'))
```

Comment:
ping/ping is more appropriate here; it's better not to have tests which modify data

Reply:
@mnunberg 
Great point!